### PR TITLE
MY-23238 use Accept: application/vnd.shipment_label_link+json to hand…

### PR DESCRIPTION
**Pull request by reindert-vetter**
handle multiple labels

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2311,36 +2311,44 @@ <h3 id="6_F_1">6.F.1 Overview </h3>
                     returned.
                     <br><br>
 
-                    If you want to retrieve large numbers of labels in one response, you can use the v2 endpoint.
-                    Because it concerns a large quantities of labels, it may happen that the PDF file takes longer to receive.
+                  If you want to retrieve more than 25 labels in one response, you can use <b>Accept: application/vnd.shipment_label_link+json; charset=utf8</b>.
+                  You will receive a url to a label, but the label is not immediately available and will return a http status of 404 until it is ready. Depending on the number of labels, this process will take between 3 seconds and 3 minutes. You can regularly check whether the label is available or you can use the <a href="#10">label created webhooks</a>.
```
Comment (EdieLemoine) [inline index.html]: URL, HTTP

File: index.html [hunk 2]
```diff
@@ -2311,36 +2311,44 @@ <h3 id="6_F_1">6.F.1 Overview </h3>
...
+                        <td>Accept: application/pdf (For the PDF binary. This is the default.)<br/>
+                          <i>For processing less than 25 labels.</i>
+                        </td>
                     </tr>
                     <tr>
-                        <td>Accept: application/json; charset=utf8 (For <a href="#7_R">ShipmentLabelDownloadLink</a>)</td>
+                        <td>Accept: application/json; charset=utf8 (For <a href="#7_R">ShipmentLabelDownloadLink</a>)<br/>
+                          <i>For processing less than 25 labels.</i>
+                        </td>
+
```
Comment (EdieLemoine) [inline index.html]: entertje

**Review by EdieLemoine — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (reindert-vetter) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (reindert-vetter) [reply index.html]: done

**Review by RichardPerdaan — APPROVED** [review]

**Pull request merged**